docs/api/instruments/racal/racal1992.rst

```diff
@@ -1,5 +1,5 @@
 ######################################
-Racal-Dana 1992 Multimeter
+Racal-Dana 1992 Universal Counter
 ######################################
```

Comment:
Could you adjust the line lengths of the lines above/below the name (i.e. the hashes) to the length of the name, please?

All three lines should have the same length.

Thanks.